src/style/button.js

```diff
@@ -17,6 +17,10 @@ const buttonStyle = {
     textDecoration: 'none',
```

Comment:
In button this transition is sort of giving me feeling of delay when I click.... coz click is such a fast action.